src/applications/proxy-rewrite/teamsite-proxy.js

```diff
@@ -0,0 +1,96 @@
+const express = require('express');
+const fetch = require('node-fetch');
+const path = require('path');
+const cors = require('cors');
+
+const BUCKETS = require('../../site/constants/buckets');
+const ENVIRONMENTS = require('../../site/constants/environments');
+const HOSTNAMES = require('../../site/constants/hostnames');
+
+const VETS_WEBSITE_URL = process.env.VETS_WEBSITE_URL || 'localhost:3001';
+const TEAMSITE_PROXY_HOST = process.env.TEAMSITE_PROXY_PORT || 'localhost';
+const TEAMSITE_PROXY_PORT = process.env.TEAMSITE_PROXY_HOST || 3500;
```

Comment:
Env variables passed from CI, https://github.com/department-of-veterans-affairs/vets-website/pull/13100/files#diff-5e5476fea1f1814a43542e706126d91eR2